.github/workflows/manual-deploy-testnet-l2.yml

```diff
@@ -591,6 +637,4 @@ jobs:
           curl -XPOST -H "Authorization: Bearer ${{ secrets.GH_TOKEN }}" -H "Accept:application/vnd.github" -H "Content-Type:application/json" https://api.github.com/repos/ten-protocol/ten-test/dispatches --data '{ "event_type": "dev_testnet_deployed", "client_payload": { "ref": "${{ github.ref_name }}" }'
 
       - name: 'Send a repository dispatch to obscuro-test on deployment of testnet'
-        if: ${{ (github.event.inputs.testnet_type == 'uat-testnet') }}
-        run: |
-          curl -XPOST -H "Authorization: Bearer ${{ secrets.GH_TOKEN }}" -H "Accept:application/vnd.github" -H "Content-Type:application/json" https://api.github.com/repos/ten-protocol/ten-test/dispatches --data '{ "event_type": "uat_testnet_deployed", "client_payload": { "ref": "${{ github.ref_name }}" }'
\ No newline at end of file
+        if: ${{ (github.event.inputs.testnet_type == 'uat-testnet') }}
```

Comment:
this looks like a section has been removed accidently? @tudor-malene

Reply:
ah yeah